test/e2e/external_gateways.go

```diff
@@ -18,7 +20,11 @@ import (
 
 // This is the image used for the containers acting as externalgateways, built
 // out from the e2e/images/Dockerfile.frr dockerfile
-const externalContainerImage = "quay.io/fpaoline/ovnkbfdtest:0.2"
+const (
+	externalContainerImage = "quay.io/fpaoline/ovnkbfdtest:0.2"
```

Comment:
Is there not a better repo for housing OVN-K CI images

Reply:
This came out when I created the original one (now I am only adding a new constant here) and we agreed with @trozet that it was fine this way. I am providing the dockerfile here so it should be easy to rebuild it if my quay repo dies for some reason.